### DISCUSSION TITLE
Scenario: Upgrading modpack to new version of Minecraft

### DISCUSSION THREAD

**Discussion by ejm554** [gorilla-devs/ferium#222] [Q&A] [answered]
Hello! I'm an amateur modpack builder and server owner, mostly making play things for small groups. I just discovered this new tool, and I'm excited to try it. 

Is it possible to use the tool to examine a set of mods meant for one version of Minecraft, e.g., 1.16.5, and see if those mods will work in a different version of Minecraft, e.g., 1.18.2? As it stands now, when I want to see if/how my modpack will work with a newer version of Minecraft, I go through the painful process of visiting the 100+ CurseForge pages to see if a mod exists for a that version.

I briefly reviewed the content of this GitHub repo/org, and I didn't see this scenario noted anywhere. It's definitely possible that it *is* mentioned, and that I somehow missed it.

**Comment by theRookieCoder** [comment, marked as answer]
After adding the list of mods that you know are working to a 1.16.5 profile, you can use `ferium profile configure` to switch the game version to 1.18.2. Then try upgrading again and you can see which mods fail to resolve to a version. (the `--dry-run` feature described in #162 would be useful for this)
